backend/globaleaks/handlers/user/reset_password.py

```diff
@@ -134,13 +136,15 @@ def validate_password_reset(session, reset_token, recovery_key, auth_code):
                 recovery_key = recovery_key.replace('-', '').upper() + '===='
                 recovery_key = Base32Encoder.decode(recovery_key.encode())
                 prv_key = GCE.symmetric_decrypt(recovery_key, Base64Encoder.decode(user.crypto_bkp_key))
-        except:
+        except Exception as e:
+            logging.debug(e)
             return {'status': 'require_recovery_key'}
 
     if user.two_factor_secret:
         try:
             State.totp_verify(user.two_factor_secret, auth_code)
-        except:
+        except Exception as e:
+            logging.debug(e)
```

Comment:
SECURITY CONCERN: HIGH.
While it is known that the silencing (pass) of an exception is typically not good practice, here  in this specific context it is very very risky to log the exception even for debugging purposes since one could end logging permanently a security token valid to impersonate a user.

Code to be removed restoring the original.